packages/connect/e2e/jest.config.ts

```diff
@@ -1,7 +1,7 @@
 export default {
     rootDir: './',
     moduleFileExtensions: ['ts', 'js'],
-    modulePathIgnorePatterns: ['node_modules'],
+    modulePathIgnorePatterns: ['node_modules', '__mocks__'],
```

Comment:
sooo, jest serches for any __mocks__ folder wherever and applies it? just thinking if this couldn't be preventing future us from using some mocks in packages/connect ?